endpoints/oauth/login.py

```diff
@@ -297,6 +301,26 @@ def attach_func():
         if result.error_message is not None:
             return _get_response(result)
 
+        # Conduct RedHat Export Compliance if enabled
+        if features.EXPORT_COMPLIANCE:
+            logger.debug("Attempting to hit export compliance service")
+            result = requests.post(
```

Comment:
@jonathankingfc is there a timeout here? I think the default action is to wait indefinitely